src/qt/textctrl.cpp

```diff
@@ -14,19 +14,364 @@
 #include "wx/qt/private/winevent.h"
 #include "wx/qt/private/utils.h"
 
+#include <algorithm>
```

Comment:
Not sure if this is really needed?

Reply:
It's not needed. A vestige of whatever work was being done at the time. Removed.